proposals/csharp-12.0/collection-expressions.md

```diff
@@ -874,6 +874,67 @@ However, given the breadth and consistency brought by the new literal syntax, we
   Span<int> span = __result;
   ```
 
+### Should *collection expression conversion* require availability of a minimal set of APIs for construction?
```

Comment:
It may be good to include a link to the recent LDM discussion on that topic, for context: https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-10-02.md#collection-expressions

Reply:
> It may be good to include a link to the recent LDM discussion on that topic

Added